http-client/src/test/java/no/nav/familie/http/ecb/ECBClientTest.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.http.ecb
+
+import no.nav.familie.http.config.ECBRestTemplate
+import no.nav.familie.http.ecb.domene.exchangeRateForCurrency
+import no.nav.familie.http.interceptor.ECBRestClientInterceptor
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.time.LocalDate
+import java.time.YearMonth
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ECBClientTest {
```

Comment:
Observasjon:
Ser at dette er en faktisk integrasjonstest som går opp mot ecb for hver gang den kjøres.
Dette betyr vel også at vi er avhengig av at ECB er oppe for å bygge koden vår (ikke at ecb er nede ofte 😛)

De andre testene i denne pakken ser jeg setter opp wiremock istedenfor å gå opp mot faktisk tredjepart, så litt usikker om man har hatt sterke regler på hva testene bør gjøre. Uansett ingen sterke meninger fra meg, men kanskje greit å dobbelsjekke med noen på dette.

Reply:
Ja, er kanskje ikke bra at testene faktisk kjører rest-kallet mot ECB. Ser litt på wiremock!